### PR TITLE
ARROW-5094: [Packaging] Add APT/Yum verification scripts

### PR DESCRIPTION
**Pull request by kou**

### CODE REVIEW

**Comment by kszucs** [comment]
Really nice!

Filed two issues for validating wheels and conda packages:
- [ARROW-5105: [Packaging] [Python] Add Wheel verification scripts](https://issues.apache.org/jira/browse/ARROW-5105)
- [ARROW-5106: [Packaging] [C++/Python] Add conda package verification scripts](https://issues.apache.org/jira/browse/ARROW-5106)

File: dev/run_docker_compose.sh [hunk 1]
```diff
@@ -25,17 +25,17 @@ ARROW_SRC=$(realpath "${PWD}/..")
 : ${DOCKER_COMPOSE_CONF:="${ARROW_SRC}/docker-compose.yml"}
 
 docker_compose() {
-  ${DOCKER_COMPOSE} -f ${DOCKER_COMPOSE_CONF} $@
+  ${DOCKER_COMPOSE} -f ${DOCKER_COMPOSE_CONF} "$@"
```
Comment (kszucs) [inline dev/run_docker_compose.sh]: I think We should get rid of this script, I don't see any advantages it adds to plain docker-compose commands.
Docker-compose automatically searches for `docker-compose.yml` upwards in the folder hierarchy, thus it finds `arrow/docker-compose.yml` properly.

**Comment by kszucs** [comment]
`BINTRAY_REPOSITORY=apache/arrow TEST_DEFAULT=0 TEST_APT=1 dev/release/verify-release-candidate.sh binaries 0.13.0 4` fails, probably because the RC packages were not updated.

Created an issue to improve the verification script:
[ARROW-5107: [Release] Validate non-RC source and binary artifacts](https://issues.apache.org/jira/browse/ARROW-5107)

**Review by kszucs — APPROVED** [review]
Great feature @kou!

File: dev/run_docker_compose.sh (hunk 1 above)
Comment (kou) [reply dev/run_docker_compose.sh]: Could you open a JIRA issue for this?
I'm not sure whether this is useful for us or not.

**Pull request closed without merging**